packages/devp2p/src/dpt/server.ts

```diff
@@ -204,12 +203,12 @@ export class Server extends EventEmitter {
       case 'pong': {
         let rkey = info.data.hash.toString('hex')
         const rkeyParity = this._parityRequestMap.get(rkey)
-        if (isTruthy(rkeyParity)) {
+        if (rkeyParity !== undefined) {
```

Comment:
Same as above, since `rkeyParity` is type `string`, would it make sense to check `typeof rparity === 'string'`?

Reply:
Done